asdf/tags/core/tests/test_ndarray.py

```diff
@@ -368,7 +368,7 @@ def test_string(tmpdir):
 
 def test_string_table(tmpdir):
     tree = {
-        'table': np.array([(b'foo', 'სამეცნიერო', 42, 53.0)])
+        'table': np.array([(b'foo', 'სამეცნიერო', '42', '53.0')])
```

Comment:
numpy is deprecating the behavior that converts these non-string values to strings